pkg/services/navtree/navtreeimpl/navtree.go

```diff
@@ -81,9 +81,9 @@ func (s *ServiceImpl) GetNavTree(c *contextmodel.ReqContext, prefs *pref.Prefere
 	hasAccess := ac.HasAccess(s.accessControl, c)
 	treeRoot := &navtree.NavTreeRoot{}
 
-	treeRoot.AddSection(s.getHomeNode(c, prefs))
+	//treeRoot.AddSection(s.getHomeNode(c, prefs))
 
-	if hasAccess(ac.EvalPermission(dashboards.ActionDashboardsRead)) {
+	if hasAccess(ac.EvalPermission(dashboards.ActionDashboardsRead)) && false {
```

Comment:
Maybe we could make a `const NI_DISABLE = false` shared variable to use for these so it's easier from scanning to know what we are disabling?